dashboard/app/views/home/_privacy.md

```diff
@@ -122,13 +122,11 @@ A pixel tag (also known as a “clear GIF” or “web beacon”) is a tiny imag
+ 
+We may allow third-party service providers to place and read their own cookies, web beacons, and similar technologies to collect information through the Website to perform the service we’ve requested. This technical information is collected directly and automatically by these third parties. Some third-party websites, such as the Code.org Facebook page and Twitter page, may be co-branded with our name or logo. Use of these third-party services are completely optional and typically meant only for adult users.  These services are governed by the privacy policy of these third parties. We encourage you to review these third-party privacy policies for details about how they may collect, store, and use your Personal Information if you choose to engage with them.
+ 
+Code.org does not allow advertising on our Website, and we do not have the ability to collect your web search history across third-party Internet websites or search engines. However, if you navigate to the Code.org Website via a web search, your web browser may automatically provide to us the web search term you used in order to find Code.org. Because Code.org doesn’t display advertising or track browsing on third party sites, we do not do anything different in response to “do not track” signals transmitted by web browsers. We use the embedded YouTube player in Privacy Enhanced Mode to deliver computer science videos within our curriculum. This means that YouTube does not place cookies or track your viewing behavior for advertising purposes. Our student-facing curriculum does not embed any YouTube videos that are not part of our curriculum. We have disabled the “rel” functionality of YouTube, which prevents the embedded YouTube player from playing related content outside of our curriculum. We have also tagged our “YouTube” videos on the site for “child-directed treatment”. Schools can choose to block access to YouTube, in which case we use a fallback option that plays videos directly from our Website. We also offer clearly marked links to visit Facebook or Twitter to share various things such as your creations or certificates on Code.org. Using an account on these third party social networking services is entirely at your option and under your control, and will not result in behavioral tracking of your browsing behavior on Code.org. However, because we know that these services use behavioral targeting as part of their advertising business model, on student-facing course and activity pages, we don’t offer links to these services to Students under the age of 13, or in schools that have blocked Internet access to these services. (To opt out of various forms of internet advertising by third parties, you may visit the following links: http://www.networkadvertising.org/choices or http://www.aboutads.info/choices/. If you use these tools, you may need to opt out separately for each of your devices and for each web browser on each device.)
```

Comment:
"third party" is hyphenated elsewhere

Reply:
thanks! I've fixed this grammar everywhere where it occurs